### PR TITLE
add list of team members page to handbook

### PR DESCRIPTION
**Pull request by sqs**
Several teammates have asked for our website to list all Sourcegraph teammates (not just me and Beyang as at https://about.sourcegraph.com/about).

I've started the list here with me and Beyang because we already had bios written.  

---

## Check your name off when you've added your section

Please add yourself using the template at the top of the newly added file. **⭐⭐ See [How to add yourself to the team members page from the pull request](https://www.youtube.com/watch?v=MBgURqD4HMM) for a quick video walkthrough on how to add yourself (if you aren't familiar with GitHub pull request editing).**

I will be responsible for ensuring all teammates have added their sections. I may merge before everyone has done so, and I'll follow up with other teammates to ensure they add their section.

(GitHub is partially down right now, so I can't autocomplete everyone's usernames, but I'll add the full list later.)

- [x] @sqs
- [x] @beyang
- [x] @christinaforney
- [x] @nicksnyder
- [ ] @dadlerj
- [x] @mercadon
- [x] @slimsag
- [x] @keegancsmith
- [x] cdf
- [ ] @mrnugget
- [ ] @tsenart
- [ ] phu
- [x] Eric F
- [x] @unknwon
- [ ] @attfarhan
- [x] @rvantonder
- [ ] @felixfbecker
- [ ] @eseliger 
- [x] @vanesa
- [ ] @ryan-blunden
- [x] Ryan S
- [ ] @kzh 
- [ ] @lguychard
- [x] Eric B-M
- [x] @uwedeportivo 
- [ ] @ggilmore
- [ ] Julia G
- [ ] @chrismwendt
- [ ] @hadrian-git 
- [ ] Aileen A
- [ ] Adam F 

@sourcegraph/team

### CODE REVIEW

File: company/team/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Quinn lives just north of the Golden Gate Bridge with his wife and daughter. Prior to Sourcegraph, Quinn co-founded Blend Labs, an enterprise technology company with ~500 employees dedicated to improving home lending. At Palantir Technologies, he created a technology platform to help two of the top five U.S. banks recover from the housing crisis. He was the first employee and developer at Bleacher Report after graduating from high school. Quinn graduated with a BS in Computer Science from Stanford.
+
+## Beyang Liu
+
+- CTO, Co-founder, Board of Directors
+- San Francisco, CA, USA 🇺🇸
+- [beyang@sourcegraph.com](mailto:beyang@sourcegraph.com), [@beyang](https://twitter.com/beyang), [LinkedIn](https://www.linkedin.com/in/beyang-liu-07651227)
+- Beyang Liu is CTO and co-founder of Sourcegraph. Prior to Sourcegraph, Beyang was a software engineer at Palantir Technologies, where he developed new data analysis software on a small, customer-facing team working with Fortune 500 companies. Beyang studied Computer Science at Stanford, where he published research in probabilistic graphical models and computer vision at the Stanford AI Lab and thoroughly enjoyed his compilers course.
+
```
Comment (christinaforney) [inline company/team/index.md]:
```suggestion

## Christina Forney
- Product Manager
- Redwood City, CA, USA 🇺🇸
- [christina@sourcegraph.com](mailto:christina@sourcegraph.com), [@christina4nee](https://twitter.com/christina4nee), [LinkedIn](https://www.linkedin.com/in/christinaforney)
- Christina loves riding and spending time with her two horses, Comet and Bentley. Prior to Sourcegraph, Christina was the head of product development and lead a small dev team at Platinum Performance. At Palantir Technologies, lead the Internal Tools team that was responsible for the code hosts, build systems, and artifact management systems at Palantir. Christina attended Cal Poly, San Luis Obispo where she earned a BS in Animal Science and a MS in Computer Science. 
```

File: company/team/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Christina Forney
+
+- Product Manager
+- Redwood City, CA, USA 🇺🇸
+- [christina@sourcegraph.com](mailto:christina@sourcegraph.com), [@christina4nee](https://twitter.com/christina4nee), [LinkedIn](https://www.linkedin.com/in/christinaforney)
+- Christina loves riding and spending time with her two horses, Comet and Bentley. Prior to Sourcegraph, Christina was the head of product development and lead a small dev team at Platinum Performance. At Palantir Technologies, lead the Internal Tools team that was responsible for the code hosts, build systems, and artifact management systems at Palantir. Christina attended Cal Poly, San Luis Obispo where she earned a BS in Animal Science and a MS in Computer Science.
```
Comment (mrnugget) [inline company/team/index.md]:
```suggestion
- Christina loves riding and spending time with her two horses, Comet and Bentley. Prior to Sourcegraph, Christina was the head of product development and lead a small dev team at Platinum Performance. At Palantir Technologies, she lead the Internal Tools team that was responsible for the code hosts, build systems, and artifact management systems at Palantir. Christina attended Cal Poly, San Luis Obispo where she earned a BS in Animal Science and a MS in Computer Science.
```

Comment (christinaforney) [reply company/team/index.md]: 🤦‍♀ thanks @mrnugget!

File: company/team/index.md [hunk 3]
```diff
@@ -69,4 +69,11 @@ To add yourself to this page, copy the following template, paste it at the end o
 - [keegan@sourcegraph.com](mailto:keegan@sourcegraph.com), [:octocat: keegancsmith](https://github.com/keegancsmith), [@keegan_csmith](https://twitter.com/keegan_csmith)
 - Keegan likes living in places with lots of sun and fibre internet. Prior to Sourcegraph, Keegan worked on the developer tools team at Facebook and spent too much time studying Mathematics and Computer Science. When not doing his day job, Keegan is usually out and about with his wife and son. He can also be found in a skatepark, on a squash court or on a mountain.
 
+## Full Name
```
Comment (keegancsmith) [inline company/team/index.md]: @uwedeportivo

Comment (unknwon) [reply company/team/index.md]: I updated for him.

**Comment by sqs** [comment]
I am going to merge this. I will follow up with individual people who haven't yet added their section to add it.

**Pull request merged**